Makefile

```
@@ -8,7 +8,7 @@ DATE     := $(shell date +%Y-%m-%d)
 LDFLAGS  := -s -w -X $(MODULE)/internal/build.Version=$(VERSION) -X $(MODULE)/internal/build.Date=$(DATE)
 PREFIX   ?= /usr/local
 
-.PHONY: build vet test unit-test integration-test install uninstall clean fetch_meta
+.PHONY: build vet test unit-test integration-test install uninstall clean fetch_meta gitleaks
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)